app/controllers/evaluations_controller.rb

```diff
@@ -3,5 +3,144 @@
 # Controller for evaluations CRUD actions.
 class EvaluationsController < ApplicationController
   before_action -> { authorize_user('evaluator') }
+  before_action :set_evaluation_and_submission_assignment, only: %i[save_draft mark_complete]
+
   def index; end
+
+  def show; end
+
+  def new
+    @evaluator_submission_assignment = find_evaluator_submission_assignment
+
+    if @evaluator_submission_assignment.nil? || !can_access_evaluation?
+      return redirect_to evaluations_path, alert: I18n.t("evaluations.alerts.evaluator_submission_assignment_not_found")
+    end
+
+    phase = @evaluator_submission_assignment.phase
+    @evaluation_form = EvaluationForm.find_by(phase:)
+
+    if @evaluation_form.nil?
+      return redirect_to evaluations_path, alert: I18n.t("evaluations.alerts.evaluation_form_not_found")
+    end
+
+    build_evaluation
+
+    render :new
+  end
+
+  def edit
+    @evaluation = Evaluation.find_by(id: params[:id])
+    return unauthorized_redirect unless can_access_evaluation?
+
+    render :edit
+  end
+
+  def save_draft
+    @evaluation.completed_at = nil
+    @evaluation.save(validate: false)
+
+    flash[:notice] = I18n.t("evaluations.notices.saved_draft")
+    redirect_to evaluations_path
+  rescue ActiveRecord::RecordInvalid, ActiveRecord::NotNullViolation
+    handle_save_draft_failure
+  end
+
+  def mark_complete
+    @evaluation.completed_at = Time.current
+
+    if @evaluation.save
+      flash[:notice] = I18n.t("evaluations.notices.marked_complete")
+      redirect_to evaluations_path
+    else
+      @evaluation.completed_at = nil
+      handle_mark_complete_failure
+    end
+  end
+
+  private
+
+  def set_evaluation_and_submission_assignment
+    @evaluation = find_or_initialize_evaluation
+    @evaluation.assign_attributes(evaluation_params)
+
+    @evaluator_submission_assignment = find_evaluator_submission_assignment
+
+    unauthorized_redirect unless can_access_evaluation?
+  end
+
+  def find_or_initialize_evaluation
+    if params[:id]
+      Evaluation.includes([:evaluation_criteria]).find(params[:id])
+    else
+      Evaluation.new
```

Comment:
it might be better to attach the current_user to the new record here rather than passing their ID as a hidden param from the form submission
```suggestion
      Evaluation.new(user_id: current_user.id)
```